freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/describe/DescribeFreeIpaResponse.java

```diff
@@ -62,6 +62,8 @@
     @ApiModelProperty(value = FreeIpaModelDescriptions.FREEIPA_SERVER_SETTINGS, required = true)
     private FreeIpaServerResponse freeIpa;
 
+    private Boolean availabilityStatus;
```

Comment:
I know we agreed it should be a 1 bit thing, but for the API I still prefer enums. just in case. it's easier to extend if we must.
I'm fine handling this as a boolean in the background